client/idrepo/common-ui/src/main/resources/META-INF/resources/ui-commons/css/search.scss

```diff
@@ -36,25 +35,44 @@
     float: left;
     padding: 0 3px 0px 0px;
     display: inline-block !important;
+    .form-check{
+      margin-bottom: -50px;
+    }
+    .toggle.btn-xs {
```

Comment:
as an improvement, for this kind of rules the related `.scss` style is:

```scss
.toggle {
  &.btn-xs {/*...*/}
}
```
and for this specific case I would do:
```scss
.toggle {
  &.btn {/*...*/}
}
```
instead, in order to avoid overriding a rule related to the default css grid classes (`xs`, `md` etc...).